src/helpers/ageInMonths.js

```diff
@@ -0,0 +1,15 @@
+export function calculateAgeInMonths(birthMonth, birthYear) {
+  // Get the current date
+  const currentDate = new Date();
```

Comment:
Would we be able to use the `getAgeData` in [roar-utils/utils.js](https://github.com/yeatmanlab/roar-utils/blob/main/src/utils.js) to calculate the ageInMonths?